README.md

```diff
@@ -142,35 +142,99 @@ Data are sourced from the json files with the same name, located in this repo `/
 
 ### NFT Claimer trusted backend
 
-Validate offchain data, and return a signature
+Validate offchain data, and return a payload
 
-#### Sign space
+#### Sign deploy
 
-Send a `POST` request with a wallet address, a space ID and a salt
+Sign and return the payload for the SpaceCollectionFactory contract, in order to deploy a new SpaceCollection contract
+
+Send a `POST` request with:
+
+| `keyname`       | Type           | Description                                                                                       | Example                                                                         |
+| --------------- | -------------- | ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| `address`       | Wallet address | The sender wallet address                                                                         | `0x00000000000000000000000000000000000000000000000000000000000004d2`            |
```

Comment:
Better to add real address here and spaceTreasury for Curl, will be easy to test :)

Reply:
You're supposed to put your own address here, else the transaction will fail. There is no 'real', nor correct address